nexus/db-queries/src/db/pool.rs

```diff
@@ -53,8 +53,7 @@ impl Pool {
         Pool { pool }
     }
 
-    #[cfg(test)]
-    pub fn new_failfast(db_config: &DbConfig) -> Self {
+    pub fn new_failfast_for_tests(db_config: &DbConfig) -> Self {
```

Comment:
ah, so the `#[cfg(test)]` disappeared because this crosses crate boundaries, right?

Reply:
Yeah

Reply:
Yeah, sadly.

Alternatively we could add a `testing` feature and expose this via `#[cfg(feature = "testing")]`, but that would cause build times to get worse.

Reply:
Yeah, I figured. I'm okay with limiting feature flags - exposing this API as-is is a con, but a relatively minor one in the grand scheme of things.